### PR TITLE
Alex/74 charities can view create challenges

### PR DESCRIPTION
**Pull request by alexandrashaw10**

### CODE REVIEW

**Comment by alexandrashaw10** [comment]
#70

**Comment by alexandrashaw10** [comment]
image
image

File: app/src/main/res/layout/activity_charity_login.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/etCharityEin" />
+
+    <Button
+        android:id="@+id/loginBtn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
```
Comment (ajin12) [inline app/src/main/res/layout/activity_charity_login.xml]: use value in dimens.xml

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+    <color name="colorAccent">@color/una_charcoal</color>
+
     <color name="bumbleYellow">#FFCB37</color>
     <color name="black">#000000</color>
     <color name="transparent">#00000000</color>
     <color name="scrim_center_grey">#66000000</color>
     <color name="scrim_start_black">#CC000000</color>
     <color name="white">#ffffff</color>
     <color name="light_gray">#dcdcdc</color>
+    <color name="una_orange_1">#3DFFD9B2</color>
```
Comment (ajin12) [inline app/src/main/res/values/colors.xml]: rip

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+    <color name="una_orange_2">#FFCA94</color>
+    <color name="una_orange_3">#ffba93</color>
+
+    <color name="una_gold">#C0B283</color>
```
Comment (ajin12) [inline app/src/main/res/values/colors.xml]: 😍

File: .travis.yml [hunk 4]
```diff
@@ -33,5 +33,4 @@ jobs:
         prerelease: true
         overwrite: true
         name: master
-        body: "Changes from $TRAVIS_COMMIT containing $TRAVIS_COMMIT_MESSAGE"
```
Comment (chiped) [inline .travis.yml]: can you change `debug-app.apk` to `app-debug.apk` in this file? I didn't verify the file name before 😅

**Pull request merged**